infra/aws/aws-organizations.md

```diff
@@ -63,9 +63,8 @@ description of the patterns we've adopted.
   billing data without having to give them direct access to the `org-root`
   account. *This cannot be changed once the account is created, so be sure
   to do it now.*
-* To access other accounts, users will use access keys in this account,
-  then assume a role in another account that allows access from this one.
-  See the [terraform-aws-iam-cross-acct-src](https://github.com/trussworks/terraform-aws-iam-cross-acct-src)
+* To access other accounts, we assign group policies which allow direct role assumption.
```

Comment:
Like I mention below, terraform-layout-example isn't actually a module. We use iam-user-group and iam-cross-acct-dest to accomplish this, and people can look at terraform-layout-example for *how* to do it.